column_transforms/rasgo_substring/rasgo_substring.yaml

```diff
@@ -0,0 +1,26 @@
+name: rasgo_substring
+transform_type: column
+source_code: rasgo_substring.sql
+description: |
+  This function creates a new column that contains a substring of either a fixed value or another column in your dataset.
+
+  # TODO: finish description
+arguments:
+  col_expr:
+    type: string
```

Comment:
I think this should be of type `column` right? We use that type so in the UI we prepopulate a select drop down with all columns in that dataset.

Easier than manually typing in.